### ISSUE TITLE
specific `ProcessRequirement`s need to have their `class` value spelled out in the spec

### ISSUE THREAD

**Issue by mr-c** [common-workflow-language/common-workflow-language#220]
For example, `SubworkflowFeatureRequirement`:

draft-2:
```
field	type	required	description
class	SubworkflowFeatureRequirement_class	True	Must be SubworkflowFeatureRequirement to indicate this is a SubworkflowFeatureRequirement object. Inherited from ProcessRequirement
```
draft-3 & draft-4.dev1:
```
field	type	required	description
class	string	True	 The specific requirement type.
```
The description should be `Must be SubworkflowFeatureRequirement` or somesuch. Likewise for the remaining FeatureRequirements.

**Comment by mr-c** [comment]
This includes #216

**Comment by mr-c** [comment]
Fixed in #248
